.travis.yml

```diff
@@ -9,7 +9,7 @@ script:
   - npm --silent run deploy-status -- --status pending --message 'Waiting for build'
 
   - npm --silent run eslint
-  - npm --silent run prettier-check
+#  - npm --silent run prettier-check
```

Comment:
Remember, don't uncomment this in the commit where you run prettier. It needs to be a separate commit _after_ the prettier run commit.

Reply:
Yep, on it.  Thanks for the reminder.